cocotb/share/lib/gpi/GpiCommon.cpp

```diff
@@ -168,32 +167,27 @@ static void gpi_load_libs(std::vector<std::string> to_load)
 
         std::string arg = *iter;
 
-        std::string lib_name;
-        std::string func_name;
-        auto it = std::find(arg.begin(), arg.end(), ':');
-        bool const has_colon = it == arg.end();
-        if (has_colon) {
-            // no colon in the string, default
-            lib_name = arg;
-            func_name = arg + "_entry_point";
-        }
-        else {
-            lib_name = std::string(arg.begin(), it);
-            func_name = std::string(it+1, arg.end());
+        auto const it = std::rfind(arg.begin(), arg.end(), ':');  // find from right since path could contain colons (Windows)
+        if (it == arg.end()) {
```

Comment:
```suggestion
        auto const idx = arg.rfind(':');  // find from right since path could contain colons (Windows)
        if (idx == std::string::npos) {
```